recipes/keycloack-gatekeeper/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set goname = "github.com/keycloak/keycloak-gatekeeper" %}
+{% set version = "2.3.0" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/", "\\" if win else "/") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ goname}}/archive/v{{ version }}.tar.gz   # [unix]
```

(1/2) It looks like keycloak-gatekeeper can be built using .tar.gz for Windows and Linux as well. This is preferable and it means the auto-tick bot will help you keep the package up-to-date. 
```suggestion
    url: https://{{ goname}}/archive/v{{ version }}.tar.gz
```